mssql-plugin/src/main/java/io/cdap/plugin/mssql/SqlServerPostAction.java

```diff
@@ -60,6 +62,68 @@ public SqlServerPostAction(SqlServerQueryActionConfig sqlServerQueryActionConfig
     @Nullable
     public Integer queryTimeout = -1;
 
+    @Name(SqlServerConstants.AUTHENTICATION)
+    @Description("Indicates which SQL authentication method will be used for the connection. Use '" +
```

Comment:
Are these descriptions the same everywhere? If so, it's probably better to also move them into SqlServerConstants.

Reply:
Moved into `SqlServerConstants`.